src/components/Tags.js

```diff
@@ -1,16 +1,19 @@
 import React from "react";
+import { Link } from "react-router-dom";
 
 export default function Tags({ tags = [] }) {
   return (
     <>
-      <div className="py-4 text-left mb-4 opacity-50">
+      <div className="py-4 text-left mb-4">
         {tags.map((tag) => (
-          <div
-            key={tag}
-            className="inline-block px-3 rounded mr-4 mb-4 border border-solid border-secondary"
-          >
-            {tag}
-          </div>
+          <Link data-e2e-id="SoundList-track-url" key={tag} to={`/tag/${tag}`}>
```

Comment:
We gave `<Link>` components for our tags a `data-e2e-id="SoundList-track-url"` attribute, but I think we should give it a `data-e2e-id="tag-url"` attribute, instead.

This current setup is causing a bug in our testing suite where Cypress looks for the first element with a `data-e2e-id="SoundList-track-url"` tag, and we expect that element to have an `href` with `"/sound/123"`, since we expect it to be a link to a track, but that `href` is `/tag/pack`, instead, because of this line. What do you think?